libs/components/src/lib/card/live-market/live-price.tsx

```diff
@@ -0,0 +1,53 @@
+import { Text } from '@deriv/quill-design';
+import clsx from 'clsx';
+import { MarketStatus } from '../types';
+
+export interface LivePriceProps {
+  status: MarketStatus;
+  bidPrice: string;
+  askPrice: string;
+}
+
+export const LivePrice = ({ status, bidPrice, askPrice }: LivePriceProps) => {
+  const colorVariant = {
+    up: 'text-solid-emerald-900',
+    down: 'text-solid-cherry-700',
+    remain: 'text-opacity-black-400',
+    closed: 'text-opacity-black-300',
+  };
+
+  const PriceContent = (price: string, status: MarketStatus) => {
+    return (
+      <div className="flex flex-row items-center gap-[2px]">
+        <Text size="md" className={clsx('pt-[6px]', colorVariant[status])}>
+          {price?.substring(0, 4)}
+        </Text>
+        <Text size="xl" className={clsx(colorVariant[status])}>
+          {price?.substring(4, 6)}
+        </Text>
+        <Text size="md" className={clsx('pb-[6px]', colorVariant[status])}>
+          {price?.substring(6)}
+        </Text>
```

Comment:
Later we need to have a way to add configuration on this.  enlargement of the numbers will vary on the instrument type.